README.md

```diff
@@ -173,6 +195,32 @@ kubectl get route -n istio-system
 
 #### Verifying the Model Registry REST Service
 
+##### Verifying OAuth Proxy Sample
+If using an OAuth Proxy sample in a non-OpenShift cluster or for using a custom service certificate,
+create a tls secret using the commands:
+
+```shell
+kubectl create secret tls modelregistry-sample-oauth-proxy --cert=path/to/cert/file --key=path/to/key/file
+```
+
+Verify that the Route was created using the command:
+
+```shell
+kubectl get routes.route.openshift.io modelregistry-sample-https
+```
+
+Use the following command to get the OpenShift Route Host:
+
+```shell
+export ROUTE_HOST=`kubectl get routes.route.openshift.io modelregistry-sample-https -ojsonpath='{.status.ingress[0].host}'`
+```
+
+Then verify the REST service using the command:
+
+```shell
+curl http://$ROUTE_HOST/api/model_registry/v1alpha3/registered_models
```

Comment:
this should use `https` rather than `http`, and an auhorization header should also be provided (`-H "Authorization: Bearer <token>"`)

Reply:
Good catch @lugi0 , fixed the curl and added a note about non-public CA.